apps/frontend/src/components/global/ExportSplunkModal.vue

```diff
@@ -38,26 +38,40 @@
             absolute="absolute"
             :value="errorCount >= 3 || errorCount < 0"
           >
-            <div class="text-center">
+            <div class="text-left">
               <p>
                 <span v-if="errorCount > 0">
                   It seems you may be having trouble using the Splunk toolkit.
                   Are you sure that you have configured it properly?
                 </span>
                 <br />
                 <span>
-                  For installation instructions and further information, check
-                  here:
+                  Accessing a Splunk instance from Heimdall requires the input
+                  of the following information:
+                  <br />
+                  username: A qualified username recognized by the referenced
+                  Splunk instance.
+                  <br />
+                  password: A qualified password recognized by the referenced
```

Comment:
Question: is there anyway to use an apikey/auth token or something with splunk as opposed to having to use hard credentials?

Reply:
Yes, I believe authentication tokens are supported as well.